src/frontend/view/view-grid.ts

```diff
@@ -75,9 +75,46 @@ const createAgGrid = () => {
     ...(sortConfig[field] || {}),
   }))
 
+  const suppressKeyboardEvent = (
+    params: SuppressKeyboardEventParams
+  ): boolean => {
+    const KEY_TAB = 'Tab'
+    const key = params.event.key
+
+    if ([key, key.toUpperCase()].includes(KEY_TAB)) {
+      return true
+    }
+
+    return false
```

Comment:
nit: wouldn't this be equivalent?
```suggestion
    return params.event.key.toUpperCase() === 'TAB'
```